app/controllers/rent_controller.rb

```diff
@@ -0,0 +1,15 @@
+class RentController < ApplicationController
+  include Wor::Paginate
+  include DeviseTokenAuth::Concerns::SetUserByToken
+
+  def create
```

Comment:
In this end point you miss 2 important things: 
1. explicit you `render`
2. handle case were body request params are invalid and the record is not created.